Sources/Fuzzilli/Configuration.swift

```diff
@@ -23,6 +23,28 @@ public struct Configuration {
     /// Used to verify that crashes can be detected.
     public let crashTests: [String]
 
+    /// Code snippets that cause an observable difference of output
+    /// in the target engine. Used to verify that crashes can be detected.
+    public let differentialTests: [String]
+
+    /// Code snippets that must not cause an observable difference of output
+    /// in the target engine. Used to verify that common sources of
+    /// entropy (Math.random, ...) are deterministic.
+    public let differentialTestsInvariant: [String]
+
+    /// During differential fuzzing, this is the rate of variables probed
+    /// at the end of the generated programs.
+    public let differentialRate: Double
```

Comment:
What's your impression of how important these are? Aren't they basically a subset of the "weaved" hash operations?
I think the name is a bit confusing. Intuitively, I'd for example think that this is the rate of differential tests being performed during fuzzing, i.e. if this is 0.5, half of the generated programs would be for differential testing, the other half would be "regular" samples. Maybe they should be called something like "differentialOutOfLineHashRate" and "differentialInlineHashRate" or something along these lines?

Reply:
I'll rename the variables/parameters as you suggested. Do you think there should be an additional variable controlling the rate of programs which are differentially tested?